stack/input/inputbase.class.php

```diff
@@ -1485,7 +1492,7 @@ public function maxima_to_response_array($in) {
     public function replace_validation_tags($state, $fieldname, $questiontext) {
 
         $name = $this->name;
-        $feedback = $this->render_validation($state, $fieldname);
+        $feedback = $this->render_validation($state, $fieldname, null);
```

Comment:
Maybe a comment here to tell why we use `null` as `$lang`; it makes sense to me that as this path is always executed with a sensible language, we do not need to override it and do that switching around, but why might not be so clear to everyone. Also are we absolutely sure that PHP syntax will never change so that the ifs on line 1322 and 1373 above will not start thinking null as true, maybe those ifs should check $lang !== null`... Same for all the other checks in all the other input types.